aspnetcore/tutorials/signalr.md

```diff
@@ -90,76 +87,85 @@ At the end, you'll have a working chat app:
 
 ## Add the SignalR client library
 
-The SignalR server library is included in the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app). But you have to get the JavaScript client library from [npm, the Node.js package manager](https://www.npmjs.com/get-npm).
+The SignalR server library is included in the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app). The JavaScript client library isn't automatically included in the project. For this tutorial, you use the [Library Manager (LibMan)](xref:client-side/libman/index) to get the client libary from *unpkg*. [unpkg](https://unpkg.com/#/) is a [content delivery network](https://en.wikipedia.org/wiki/Content_delivery_network) that can deliver anything found in [npm, the Node.js package manager](https://www.npmjs.com/get-npm).
```

Comment:
- client libary --> client library
- remove the "en." segment from the Wikipedia URL